packages/react-geo/.lintstagedrc.js

```diff
@@ -0,0 +1,6 @@
+const config = require('../../.lintstagedrc.js');
+
+module.exports = {
+  ...config,
+  '*.{ts,tsx,js}': 'eslint',
```

Comment:
Just curious, do we need to lint `.js` files?

Reply:
Hmm, we do not have any .js files in Geo package.
Will remove that